dalle/dalle/dalle.py

```diff
@@ -0,0 +1,60 @@
+"""Welcome to Pynecone! This file outlines the steps to create a basic app."""
```

Comment:
I think this file needs to be linted. Later we should add those pre-commit hooks to this repo also.